### ISSUE TITLE
Color The Nodes With A Gradient

### ISSUE THREAD

**Issue by ilijagjorgjiev** [ilijagjorgjiev/RESTfulConversationMining#13]
Color each node with a gradient color depending on how many times it was called.

**Comment by ilijagjorgjiev** [comment]
Note: 
1. When there are multiple statuses for the same request, the key request node should have the total length.
2. Don't color the XOR.
3. Enabling and disabling functions for the css.
4. Color the edges the same way as the node is colored. 
5. Selection of different client/time period, that activates the css function.
6. Css function should work also if selected for everything.

**Comment by ilijagjorgjiev** [comment]
Please refactor using the class in the nodes and the injected stylesheet which can be activated/deactivated

**Comment by ilijagjorgjiev** [comment]
@pautasso This was done as we discussed and started in the meeting, with one small exception each time I draw a new graph, I delete the previous stylesheet and create a new one, since we may have more gradient colors or less, this actually makes a problem. 
This may be solvable by actually disabling some classes if they are not necessary, but I decided to go for this option that I explained.

Also, enable/disable buttons were added.
